Frontend/ui-driver/src/Flow.purs

```diff
@@ -3237,10 +3237,7 @@ referralFlow :: FlowBT String Unit
 referralFlow = do
   appConfig <- getAppConfigFlowBT Constants.appConfig
   let cityConfig = getCityConfig appConfig.cityConfig (getValueToLocalStore DRIVER_LOCATION)
-      referralType = case (cityConfig.showDriverReferral || appConfig.enableDriverReferral), (cityConfig.showCustomerReferral || appConfig.enableCustomerReferral) of
```

Comment:
if we removed this, there's no requirement of other two variables - we should remove redundant code as well

Reply:
please create a separate PR for this, currently merging as it's needed